mpegts/common.cpp

```diff
@@ -25,7 +25,7 @@ void writePts(SimpleBuffer &rSb, uint32_t lFb, uint64_t lPts) {
 
 uint64_t readPts(SimpleBuffer &rSb) {
     uint64_t lPts = 0;
-    uint32_t lVal = 0;
+    uint64_t lVal = 0;
     lVal = rSb.read1Byte();
     lPts |= ((lVal >> 1) & 0x07) << 30;
```

Comment:
Here we want to shift 3 bits to position 33-31, but lVal was a 32 bit integer, hence the highest bit was discarded.